tests/Enterspeed.Delivery.Sdk.Tests/Domain/Models/DeliveryQueryBuilderTests.cs

```diff
@@ -65,7 +66,7 @@ public void Url_CanAddUrl_Contains()
             var query = sut.Build();
             var uri = query.GetUri(new Uri("https://example.com"), "/v1");
 
-            Assert.Contains("url=/test/", uri.AbsoluteUri);
+            Assert.Contains("url=/test/", HttpUtility.UrlDecode(uri.AbsoluteUri));
```

Comment:
@jthind is this fair? The test failed as it now creates Urls that would look like: http://example.com?url=%2Ftest%2F